.readme-partials.yml

```diff
@@ -0,0 +1,27 @@
+body: |-
+        ### Using TypeScript
+
+        `@google-cloud/cloudbuild` provides TypeScript type definitions, and can
+        be used inside a TypeScript project:
+
+        ```js
+        import {CloudBuildClient} from '@google-cloud/cloudbuild';
+        async function quickstart(
+          projectId = 'YOUR_PROJECT_ID', // Your Google Cloud Platform project ID
```

Comment:
If we _did_ decide to embed a sample, we'd want to follow the rubric guidelines more closely.  The parameters here would be passed into a wrapping `main` function, and have commented out assignments.